LibraBFT/ImplShared/Consensus/Types/EpochIndep.agda

```diff
@@ -561,6 +560,9 @@ module LibraBFT.ImplShared.Consensus.Types.EpochIndep where
   unquoteDecl vciPublicKey   vciVotingPower = mkLens (quote ValidatorConsensusInfo)
              (vciPublicKey ∷ vciVotingPower ∷ [])
 
+  data ObmNotValidProposerReason : Set where
+    ProposalDoesNotHaveAnAuthor ProposerForBlockIsNotValidForThisRound NotValidProposer : ObmNotValidProposerReason
+
```

Comment:
no big deal, but could this be formatted:

```
data ObmNotValidProposerReason : Set where
    ProposalDoesNotHaveAnAuthor
    ProposerForBlockIsNotValidForThisRound 
    NotValidProposer 
    : ObmNotValidProposerReason
```

I say this because when I initially looked at the code my eyes only saw the first constructor and I thought, if there was only going to be one, then it should be a different one.

Main point : easier visual pattern matching against Haskell code.